pkg/agent/proxy/proxier.go

```diff
@@ -542,35 +571,47 @@ func (p *proxier) installServices() {
 			}
 
 			// Install ClusterIP flows for the Service.
-			groupID := p.groupCounter.AllocateIfNotExist(svcPortName, internalPolicyLocal)
+			groupID, exists := p.groupCounter.Get(svcPortName, internalPolicyLocal)
+			if !exists {
+				klog.InfoS("Group for Service internalTrafficPolicy is not installed", "ServiceName", svcPortName, "ServiceInfo", svcInfoStr, "internalTrafficPolicy", internalPolicyLocal)
```

Comment:
This should be an Error. And log should use past tense. "was not installed"

Reply:
Thanks, updated.